routes/api/questions.js

```diff
@@ -0,0 +1,59 @@
+"use strict";
```

Comment:
You will also need to edit routes/api/index.js to require this file and connect it to the `/questions` path... follow the pattern of what you see in the file...